src/structures.jl

```diff
@@ -1,7 +1,14 @@
-export GraphAssetData, GraphFlowData, RepresentativePeriod, TimeBlock
+export GraphAssetData, GraphFlowData, RepresentativePeriod, BasePeriod, TimeBlock
 
 const TimeBlock = UnitRange{Int}
 
+"""
+Structure to hold the data of the base periods.
+"""
+struct BasePeriod
+    num_base_periods::TimeBlock
```

Comment:
Since I don't know what is coming in the next commit, this might not be applicable.
`num_base_periods` implies that this is a number. `TimeBlock` is a set. I don't know what `base_periods` will be used for, so for now I am leaving it like this.

Reply:
You are right; it is easy to handle it as a number. Thanks! I've changed it in the code.